getting-started/templates/sizing-examples/data-management-sizing-example.yaml

```diff
@@ -177,6 +177,19 @@ dataframeservice:
     optimizeQueue:
       ## @param dataframeservice.workloadManagement.optimizeQueue.concurrencyLimit Number of parallel jobs processed from the queue.
       concurrencyLimit: 15
+    # Dremio targets high-cost queries to this queue, such as decimated queries
```

Comment:
Suggestion:
> Dremio moves high-cost queries, such as decimation queries that scan tables with more than a million rows, to this queue.

Reply:
I think "targets" is more correct than "moves". To me, "moves" implies that the request was somewhere else before it entered the queue, which isn't conceptually correct. Another option would be 'routes". I updated with the other suggested syntax.